src/Components/Item.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './item.css';
+
+const Item = (state) => {
+  const {
+    id, title, price, image,
+  } = state;
+
+  const UNITPRICE = 'USD';
+  return (
+    <div className="products">
+      <div className="contain" key={id}>
+        <img
+          className="images"
+          src={image}
+          alt="Nice cool products"
+          style={{ width: '200px', height: '200px' }}
+        />
+        <div className="sub-cont">
+          <h3 className="title">{title}</h3>
+          <div className="price">{`${UNITPRICE} ${price}`}</div>
+          <Link to={`/products/${id}`}>
+            <div className="btn">see product Details</div>
```

Comment:
- [ ] The price color on the details page is not a clear view. Kindly work on it. :+1: 
![pricecolor](https://user-images.githubusercontent.com/70483835/189468709-94b44f31-1937-402d-8a17-f91bfb96feee.png)